oauth2client/client.py

```diff
@@ -614,6 +621,34 @@ def apply(self, headers):
     """
     headers['Authorization'] = 'Bearer ' + self.access_token
 
+  def has_scopes(self, scopes):
+    """Verify that the credentials are authorized for the given scopes.
+
+    Returns True if the credentials authorized scopes contain all of the scopes
+    given.
+
+    Args:
+      scopes: list or string, the scopes to check.
+
+    Notes:
+      There are cases where the credentials are unaware of which scopes are
+      authorized. Notably, credentials obtained and stored before this code was
+      added will not have scopes, AccessTokenCredentials do not have scopes. In
+      both cases, you can use refresh_scopes() to obtain the canonical set of
+      scopes.
+    """
+    scopes = util.string_to_scopes(scopes)
+    return set(scopes).issubset(set(self.scopes))
```

Comment:
@anthmgoogle why a space separated string? 

I was able to modify ``_to_json`` to work with a set, but modifying it to do a space separated string might be more complicated than it's worth.